bolt/src/main/java/com/slack/api/bolt/service/builtin/DefaultOAuthCallbackService.java

```diff
@@ -37,15 +40,45 @@ public DefaultOAuthCallbackService(
             OAuthAccessErrorHandler accessErrorHandler,
             OAuthV2AccessErrorHandler accessV2ErrorHandler,
             OAuthExceptionHandler exceptionHandler) {
+        this(
+                config,
+                stateService,
+                successHandler,
+                successV2Handler,
+                errorHandler,
+                stateErrorHandler,
+                accessErrorHandler,
+                accessV2ErrorHandler,
+                exceptionHandler,
+                (request, response, apiResponse) -> response,
```

Comment:
It has been a few years since I've done Java development but I did not know Java supports this kind of terse syntax for anonymous functions! Cool!